pocs/camera/camera.py

```diff
@@ -68,7 +68,7 @@ def __init__(self,
                     self.logger.critical("Couldn't import Focuser module {}!".format(module))
                     raise err
                 else:
-                    self.focuser = module.Focuser(**focuser, camera=self)
+                    self.focuser = module.Focuser(**focuser, camera=self, config=self.config)
```

Comment:
Note that this construct can lead to run time errors due to adding conflicting entries into the focuser dict: if it contains camera or config, then an exception will be raised. Not saying that you shouldn't use it, but it can be useful to make a copy of the dict and then insert the overrides (e.g. camera= and config=), allowing you to warn of errors instead of dying or to simply provide a better error message.

Reply:
OK, I see your point. The `focuser` `dict` _shouldn't_ ever contain `camera` or `config` items, but it would break things if it somehow did. Will combine into a `kwargs` dict before passing to the `Focuser` constructor.